src/include/sof/audio/mux.h

```diff
@@ -35,6 +35,12 @@ struct comp_dev;
 STATIC_ASSERT(MUX_MAX_STREAMS < PLATFORM_MAX_STREAMS,
 	      unsupported_amount_of_streams_for_mux);
 
+struct mux_look_up_entry {
+	uint32_t stream_id;
+	uint32_t in_ch;
+	bool set;
```

Comment:
What is this used for ?

Reply:
In order to create look up table. I utilized it mux/demux process functions.